plugins/modules/pacman_key.py

```diff
@@ -78,9 +78,9 @@
     default: /etc/pacman.d/gnupg
   state:
     description:
-      - Ensures that the key is present (added) or absent (revoked).
+      - Ensures that the key is present (added), trusted (signed and not expired) or absent (revoked).
```

Comment:
How does the PR ensure that the key is not expired? If the key provided to the module is already expired, I don't see how the module can change that.

Reply:
It can't extend keys of course, you're right. It just won't return a success status if the key is expired.

Maybe "Ensures/checks" would be clearer? How long are these descriptions supposed to be?